Random.py

```diff
@@ -46,21 +46,4 @@ def print_students(self):
 manager = StudentManager()
 manager.add_student("Alice", 20, 1)
```

Comment:
The indentation of the lines after function definitions is inconsistent. This can lead to runtime errors. ```diff -    print("Moves:", moves) -    print("Flags:", flags) -    print("Mines found:", mines_found) +    print("Moves:", moves) +    print("Flags:", flags) +    print("Mines found:", mines_found) ``` ---